### PR TITLE
Add checkbox for fullscreen popups.

### PR DESCRIPTION
**Pull request by bradtriebwasser**
Adds a checkbox which allows the user to toggle whether the opened popup opens in fullscreen or not.
Also adds label tags on the input labels.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ async function updateScreens(requestPermission = true) {
 
 function getFeaturesFromOptions(options) {
   return "left=" + options.x + ",top=" + options.y +
-         ",width=" + options.width + ",height=" + options.height;
+         ",width=" + options.width + ",height=" + options.height + (options.fullscreen ? ",fullscreen" : "");
```
Comment (michaelwasserman) [inline main.js]: optional nit: wrap onto new line

File: index.html [hunk 2]
```diff
@@ -11,12 +11,13 @@ <h3>Multi-Screen Window Placement Demo (<a href="https://github.com/michaelwasse
...
   <div id="openWindowControls" style="display:flex; align-items:center;">
-    <button id="openWindowButton">Open window</button>
-    URL: <input id="openWindowUrlInput" style="flex-grow:1;" value="./popup.html"></input>
+    <button id="openWindowButton" style="flex-shrink: 0;">Open window</button>
+    URL: <input id="openWindowUrlInput" style="flex-grow: 1;min-width: 81px;max-width: 125px;" value="./popup.html"></input>
     left: <input id="openWindowLeftInput" style="width:50px;" size=5 value="100" type="number"></input>
     top: <input id="openWindowTopInput" style="width:50px;" size=5 value="50" type="number"></input>
     width: <input id="openWindowWidthInput" style="width:50px;" size=5 value="830" type="number"></input>
     height: <input id="openWindowHeightInput" style="width:50px;" size=5 value="750" type="number"></input>
+    <label for="fullscreenPopupInput"> fullscreen: </label> <input type="checkbox" id="fullscreenPopupInput" name="fullscreenPopup" value="true"><br>
```
Comment (michaelwasserman) [inline index.html]: optional nit: `<label>` is nice! use that for other inputs here and in popup.html too (or I can easily follow up afterwards)

Comment (michaelwasserman) [inline index.html]: q: is value="true" useful? (seems unnecessary)

**Review by michaelwasserman — APPROVED** [review]
lgtm with optional minor comments

File: main.js (hunk 1 above)
Comment (bradtriebwasser) [reply main.js]: Done

Comment (bradtriebwasser) [reply main.js]: Done

File: index.html (hunk 2 above)
Comment (bradtriebwasser) [reply index.html]: Done.

Comment (bradtriebwasser) [reply index.html]: Removed.

**Review by michaelwasserman — APPROVED** [review]
Thanks!

**Pull request merged**